features/step_definitions/route.rb

```diff
@@ -137,7 +137,7 @@
     | f |  #{BushSlicer::HOME}/testdata/networking/service_with_selector.json |
   })
   step %Q/I expose the "selector-service" service/
-  step %Q/I have a test-client-pod in the project/
+  step %Q/I have a pod-for-ping in the project/
   step %Q/I execute on the pod:/, table(%{
     | bash | -c | curl http://<%= route("selector-service", service("selector-service")).dns(by: user) %>/ --connect-timeout 10 -I -v -s |
```

Comment:
`bash -c` is not a must there.
You can try `| curl | -foo | -bar | my_url |`

Reply:
Yes, you're right. Using curl is also working. 
see: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/224974/console